src/synology_dsm/api/core/external_usb.py

```diff
@@ -47,6 +47,7 @@ class SynoCoreExternalUSB(SynoBaseApi["dict[str, SynoCoreExternalUSBDevice]"]):
 
     async def update(self) -> None:
         """Updates external USB storage device data."""
+        self._data = {}
```

Comment:
```suggestion
        self._data: dict[str, SynoCoreExternalUSBDevice] = {}
```

we should keep the lib fully typed :wink: